specs/SingleAppHost.md

```diff
@@ -0,0 +1,65 @@
+# Background
+While WebViews from different app processes normally don't share the same WebView browser process instance, app processes from the same product suite may share the same webview browser process
+instance by specifying the same user data folder when creating WebView2Environment object.
+The `ExclusiveUserDataFolderAccess` property is for the developer to express the sharing intent, so that we could provide optimized security and performance according expected usage.
+
+# Description
+The `ExclusiveUserDataFolderAccess` property indicates whether other processes can create WebView2 sharing the same WebView browser process instance by using WebView2Environment created with the same user data folder.
+Default is FALSE.
+
+# Examples
+## Win32 C++
+```cpp
+    auto options = Microsoft::WRL::Make<CoreWebView2EnvironmentOptions>();
```

Comment:
Need to QI for `ICoreWebView2EnvironmentOptions_2`.

```cpp
Microsoft::WRL::ComPtr<ICoreWebView2EnvironmentOptions_2> options2;
if (SUCCEEDED(options.As(&options2)))
{
    CHECK_FAILURE(options2->put_ExclusiveUserDataFolderAccess(TRUE));
}
```

Reply:
Actually, there is no need, CoreWebView2EnvironmentOptions is a concrete class provided in SDK that implements both ICoreWebView2EnvironmentOptions and ICoreWebView2EnvironmentOptions_2.